src/test/resources/PEMExample.xml

```diff
@@ -0,0 +1,21 @@
iven><family>XXXX</family></name></assignedPerson><representedOrganization classCode="ORG" determinerCode="INSTANCE"><templateId root="2.16.840.1.113883.2.1.3.2.4.18.2" extension="COCD_TP145210GB01#representedOrganization" /><id extension="RYH" root="2.16.840.1.113883.2.1.3.2.4.19.1" /><name>NHSD 111</name></representedOrganization></assignedEntity></responsibleParty><location typeCode="LOC"><templateId root="2.16.840.1.113883.2.1.3.2.4.18.2" extension="COCD_TP146232GB01#location" /><npfitlc:contentId root="2.16.840.1.113883.2.1.3.2.4.18.16" extension="COCD_TP145222GB02#HealthCareFacility" /><healthCareFacility classCode="ISDLOC"><templateId extension="COCD_TP145222GB02#HealthCareFacility" root="2.16.840.1.113883.2.1.3.2.4.18.2" /><id nullFlavor="NA" /><location classCode="PLC" determinerCode="INSTANCE"><templateId extension="COCD_TP145222GB02#location" root="2.16.840.1.113883.2.1.3.2.4.18.2" /><name>Incident Location</name><addr use="PHYS"><streetAddressLine>XXXX</streetAddressLine><streetAddressLine>London</streetAddressLine><postalCode>SE12 XXX</postalCode></addr></location></healthCareFacility></location></encompassingEncounter></componentOf><component typeCode="COMP" contextConductionInd="true"><structuredBody classCode="DOCBODY" moodCode="EVN"><component typeCode="COMP" contextConductionInd="true"><section classCode="DOCSECT" moodCode="EVN"><id root="4A1C2DCB-5EB0-4D63-91DF-CB31C1F715DC" /><component typeCode="COMP" contextConductionInd="true"><npfitlc:contentId root="2.16.840.1.113883.2.1.3.2.4.18.16" extension="COCD_TP146246GB01#Section1" /><section classCode="DOCSECT" moodCode="EVN"><templateId root="2.16.840.1.113883.2.1.3.2.4.18.2" extension="COCD_TP146246GB01#Section1" /><id root="D7431B43-1B03-4D63-AD62-A843045D8669" /><title>Patient's Reported Condition</title><text><content>abdo pain, diarrhoea, fever, headache, fatigued</content></text></section></component><component typeCode="COMP" contextConductionInd="true"><npfitlc:contentId root="2.16.840.1.113883.2.1.3.2.4.18.16" extension="COCD_TP146246GB01#Section1" /><section classCode="DOCSECT" moodCode="EVN"><templateId root="2.16.840.1.113883.2.1.3.2.4.18.2" extension="COCD_TP146246GB01#Section1" /><id root="67366C18-6131-44D6-BADB-6B722E4CADB1" /><title>Case Summary</title><text><content styleCode="bold">Disposition:</content><br /><content>The disposition is: Home Management</content><br /><content>Dx25</content><br /><br /><content styleCode="bold">Selected care service:</content><br /><content>No referral made.</content><br /><br /><content styleCode="bold">Rationale:</content><br /><content>No Coronavirus concern or risk<br />Consultation completed with home management<br /></content><br /><br /><content styleCode="bold">Pathways Assessment:</content><br /><content>Coronavirus concern not identified.<br />The consultation was completed with home management.<br />Instructions given were: From what you have told me, the problem can safely be looked after at home.<br />Advice given:
```

Comment:
Can you please verify the content of does not belong to a real person and the data contained in it is realistic dummy data.